index.js

```diff
@@ -0,0 +1,12 @@
+const NewsModel = require('./newsModel')
+const NewsView = require('./newsView')
+
+const model = new NewsModel();
+const view = new NewsView();
```

Comment:
You need to include `model` that you created on line 4 in as the argument when creating `view` on line 5.

Without it, when I open `index.html` in the browser, it doesn't show anything because line 12 of this file (`index.js`) doesn't run properly because line 8 of `newsView.js` doesn't know what model to read.